### ISSUE TITLE
Integrate marketplace tests into npm test

### ISSUE THREAD

**Issue by cuongdo** [OriginProtocol/origin-js#373]
The marketplace tests added in #364 aren't yet integrated into `npm test`, which means they're invisible to CI. Filing this to make more visible.

@tyleryasaka, not sure if you were planning to work on this

**Comment by tyleryasaka** [comment]
@cuongdo Yeah I'll look into this. I would rather that we wait until we decide whether we're using the Truffle tests or the custom test helper. Then we can start migrating all our tests to that format and make sure `npm test` runs those tests.

I'll be looking into it.

**Comment by cuongdo** [comment]
Note that the OpenZeppelin token tests in `contracts/test/openzeppelin` are all imported from OpenZeppelin. Because OpenZeppelin is still gradually evolving their tests, it'd be nice to keep those tests in their [original form](https://github.com/OpenZeppelin/openzeppelin-solidity/tree/master/test/token/ERC20) (Truffle tests) to make merging test changes trivial.

**Comment by tyleryasaka** [comment]
@cuongdo Why are we running OpenZeppelin tests? Did we not decide not to modify the OpenZeppelin contract code? (If we're not modifying the OpenZeppelin contracts, we have no need for the tests.)

**Comment by cuongdo** [comment]
We have some additions to the contracts to allow possible future token migrations (like Augur's) :
https://github.com/OriginProtocol/origin-js/pull/371

This restricts burning of tokens to the contract owner (OpenZeppelin allows any token holder to burn tokens):
https://github.com/OriginProtocol/origin-js/pull/345/commits/8add1d7985f6aa0ca838e66215bd030d0dbb5d1a#diff-a45ccc6cc0733f203d0983e4629180caR30

Also, the OpenZeppelin token code was created in a different build environment, and their test environment doesn't test with the set of token contracts ours inherits from.

In short, there are enough differences that I'd be worried about the token code atrophying in our environment without OpenZeppelin tests running regularly. 

*However*, they don't necessarily have to run in every invocation of `npm test`. They can just run during CI, for example.

**Comment by tyleryasaka** [comment]
Resolved by https://github.com/OriginProtocol/origin-js/pull/380
